samples/funnyHats/index.html

```diff
@@ -53,72 +54,72 @@
     data-scale-factor="1.2" data-y-offset="0.25">
   <img id="hat17" class="hidden hat-image" src="resources/hats/17.png" data-name="Viking helmet" data-scale-factor="1.7"
     data-y-offset="0.2">
-  <img id="hat18" class="hidden hat-image" src="resources/hats/18.png" data-name="Mexican hat" data-scale-factor="1.2"
-    data-y-offset="0.1">
+  <img id="hat18" class="hidden hat-image" src="resources/hats/18.png" data-name="Mexican hat" data-scale-factor="1.35"
+    data-y-offset="0.2">
   <img id="hat19" class="hidden hat-image" src="resources/hats/19.png" data-name="Cat ears" data-scale-factor="1.2"
-    data-y-offset="0.4">
-  <img id="hat20" class="hidden hat-image" src="resources/hats/20.png" data-name="Brown hat" data-scale-factor="1.4"
-    data-y-offset="0.4">
-
-  <img id="glasses0" class="hidden glasses-image" src="resources/glasses/0.png" data-name="Deal with it" data-scale-factor="1.4"
-    data-y-offset="0.5">
-  <img id="glasses1" class="hidden glasses-image" src="resources/glasses/1.png" data-name="Harry Potter" data-scale-factor="1.2"
-    data-y-offset="0.4">
-  <img id="glasses2" class="hidden glasses-image" src="resources/glasses/2.png" data-name="Red sunglasses" data-scale-factor="1.5"
-    data-y-offset="0.6">
-  <img id="glasses3" class="hidden glasses-image" src="resources/glasses/3.png" data-name="Glasses with a nose" data-scale-factor="1.2"
     data-y-offset="0.3">
-  <img id="glasses4" class="hidden glasses-image" src="resources/glasses/4.png" data-name="Round black glasses" data-scale-factor="1.1"
+  <img id="hat20" class="hidden hat-image" src="resources/hats/20.png" data-name="Brown hat" data-scale-factor="1.5"
     data-y-offset="0.4">
-  <img id="glasses5" class="hidden glasses-image" src="resources/glasses/5.png" data-name="3d glasses" data-scale-factor="1.4"
-    data-y-offset="0.6">
-  <img id="glasses6" class="hidden glasses-image" src="resources/glasses/6.png" data-name="Yellow sunglasses" data-scale-factor="1.4"
-    data-y-offset="0.5">
-  <img id="glasses7" class="hidden glasses-image" src="resources/glasses/7.png" data-name="Heart glasses" data-scale-factor="1.5"
-    data-y-offset="0.7">
-  <img id="glasses8" class="hidden glasses-image" src="resources/glasses/8.png" data-name="Round white glasses" data-scale-factor="1.2"
-    data-y-offset="0.4">
-  <img id="glasses9" class="hidden glasses-image" src="resources/glasses/9.png" data-name="Blue sunglasses" data-scale-factor="1.4"
-    data-y-offset="0.5">
-  <img id="glasses10" class="hidden glasses-image" src="resources/glasses/10.png" data-name="Pink sunglasses" data-scale-factor="1.4"
-    data-y-offset="0.5">
-  <img id="glasses11" class="hidden glasses-image" src="resources/glasses/11.png" data-name="Red women's glasses" data-scale-factor="1.6"
-    data-y-offset="0.7">
+
+  <img id="glasses0" class="hidden glasses-image" src="resources/glasses/0.png" data-name="Deal with it"
```

Comment:
Just add a TODO here, so we don't forget, that we need to optimize images(size,resolution,etc) and loading in subsequent PRs .. and ofcourse check Licenses of images used ;)

Reply:
How about 
```js
let imagesPath = "resources/glasses/"
let imagesArray = ['1.png' , '2.png' , ...]

imagesArray.forEach( element => {
  let newImg = document.createElement("img");
  newImg.setAttribute("src", imagesPath + element);
  document.body.appendChild(newImg);
});
```

also `src`  attribute triggers immediate image load, maybe use `data-src` attribute with IntersectionObserver for lazy loading later on.
Let's also check [Lazy loading on Chrome natively](https://addyosmani.com/blog/lazy-loading/). Right now, its behind a flag, but I guess it should be soon ready for prime time.

Reply:
One finding about `cv.imread()`:
We can load an image only using `img` or `canvas` tag. 
It would be nice if opencv.js would support loading from a file directly.